packages/public/server/src/module/Subject/config/module.config.php

```diff
@@ -108,12 +108,12 @@
                 ],
                 'child_routes' => [
                     'entity' => [
-                        'type'    => 'segment',
+                        'type'    => 'literal',
                         'options' => [
-                            'route'    => '/entity/:action',
+                            'route'    => '/entity/trashBin',
```

Comment:
Don't use camel case in url
```suggestion
                            'route'    => '/entity/trash-bin',
```